src/lang.js

```diff
@@ -8,6 +8,42 @@
    * @words expect array
    *
    */
+  
+
+  let setLangType = function(LangType) {
+    let currentLangType;
+    //  Check language type is present in congreaLanguages or not.
+    if(window.congreaLanguages.hasOwnProperty(LangType)) {
+      currentLangType = LangType;
+    } else if(LangType.length>=4) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Operator '>=' must be spaced.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)